omod/src/main/web/dashboardwidgets/obsacrossencounters/obsacrossencounters.controller.js

```diff
@@ -145,9 +167,41 @@ export default class ObsAcrossEncountersController {
         };
         this.simpleEncs.push(tempEnc);
       }
+      this.updateWithConceptShortNames(this.simpleEncs);
     });
   }
 
+  updateWithConceptShortNames(encounters) {
```

Comment:
Let's add a comment here, "iterates through all obs, and for all value coded, replace the default obs.value.display  with the value display of the appropriate shortName"